docs/tutorials/fundamentals/tutorial-buy-uos.md

```diff
@@ -0,0 +1,50 @@
+---
+title: 'Tutorial - How to Buy UOS Tokens Using the Ultra Wallet'
+order: -99994
+outline: [0, 6]
+---
+
+# Tutorial - How to Buy UOS Tokens Using the Ultra Wallet
+
+This tutorial will guide you through the process of purchasing UOS tokens using the Ultra Wallet extension. UOS is the native token of the Ultra Network, and having it enables you to participate in various activities within the Ultra ecosystem.
+
+## Step 1: Open the Ultra Wallet Extension
+
+Begin by opening the Ultra Wallet extension in your browser. This is where all your transactions for buying UOS will initiate.
+
+![](./images/buying-UOS-open-wallet-extension.png)
```

Comment:
This image is a bit scuffed, can you please create a better looking one? Can use the one from `ultra-wallet-extension-panel.png`